ci/release/update-version.sh

```diff
@@ -61,3 +62,6 @@ for DEP in "${DEPENDENCIES[@]}"; do
     sed_runner "/\"${DEP}==/ s/==.*\"/==${NEXT_SHORT_TAG_PEP440}.*,>=0.0.0a0\"/g" "${FILE}"
   done
 done
+
+# Update CI image tags of the form {rapids_version}-{something}
+sed_runner "s/:[0-9]*\\.[0-9]*-/:${NEXT_SHORT_TAG}-/g" ./CONTRIBUTING.md
```

Comment:
@gforsyth Generally we do not add automation to `ci/release/update-version.sh` if it has no effect. What line in `CONTRIBUTING.md` does this modify?

Reply:
Yep